### PR TITLE
Expose the Cobb's Angle tool from cornerstone.

### PR DESCRIPTION
**Pull request by dbejarano32**
### PR Checklist

- [ Expose the Cobb's Angle tool from cornerstone.] Brief description of changes
- [ https://github.com/OHIF/Viewers/issues/2399 ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] `@` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by ed-morais** [comment]
Hello!
The code is implementing cobb's angle just fine, but the clear (trash) button is not deleting cobb's angle measures.
Any idea on how to solve this problem?

File: extensions/cornerstone/src/toolbarModule.js [hunk 1]
```diff
@@ -98,6 +98,15 @@ const definitions = [
     commandName: 'setToolActive',
     commandOptions: { toolName: 'Angle' },
   },
+  {
```
Comment (sedghi) [inline extensions/cornerstone/src/toolbarModule.js:101]: Multiple projects are using the master branch of the OHIF and don't need the Cobb Angle, so I recommend taking it out of the toolbar, or maybe put it in `More`

File: platform/ui/src/elements/Icon/icons/angle-open.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="578" height="763" viewBox="0 0 578 763" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (sedghi) [inline platform/ui/src/elements/Icon/icons/angle-open.svg:1]: Where is the source of this SVG? Is it allowed to be re-used without attribution?

File: extensions/cornerstone/src/toolbarModule.js (hunk 1 above)
Comment (dbejarano32) [reply extensions/cornerstone/src/toolbarModule.js:101]: good idea!

File: platform/ui/src/elements/Icon/icons/angle-open.svg (hunk 2 above)
Comment (dbejarano32) [reply platform/ui/src/elements/Icon/icons/angle-open.svg:1]: It's a SVG designed by @dbejarano32 for OHIF project and any fork of this project!

**Comment by sedghi** [comment]
base has changed, read more here https://github.com/OHIF/Viewers/issues/3477

**Comment by sedghi** [comment]
Closing in favor of https://github.com/OHIF/Viewers/pull/4127

**Pull request closed without merging**